### DISCUSSION TITLE
Add query results to an existing record set

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5638] [Feature Requests]
This would be a useful feature in the query builder.

This will be possible when adding records to a record set using the Query Builder feature in a future update.

**Comment by maxpatiiuk** [comment]
Once https://github.com/specify/specify7/issues/1665 is implemented, that can be used as a workaround for that feature

**Comment by melton-jason** [comment]
Record Sets currently support creating and running a query to bulk-add new records. 
Is this issue specifically regarding adding every query result from an already existing query?

**Comment by maxpatiiuk** [comment]
Correct.

> Record Sets currently support creating and running a query to bulk-add new records.
Not practical when you want to add a lot of records at once

**Comment by maxpatiiuk** [comment]
From @grantfitzsimmons, the chief duplicate issue creator 😃 

> When trying to print a group of collection object records at one time, it is not possible to add a selected group of items based on an **existing query** to a record set that already exists.
> 
> In Specify 6, this is solved by merging record sets together. This is not supported in Specify 7.
> 
> **Requested By:** Geneva

**Comment by grantfitzsimmons** [comment]
> From @grantfitzsimmons, the chief duplicate issue creator 😃

GitHub should use ChatGPT for their searches, then! I really tried to find this 😂

Not sure how I couldn't!
